contentcuration/contentcuration/static/js/administration/views.js

```diff
@@ -1,38 +1,60 @@
 var Backbone = require("backbone");
-var Models = require("edit_channel/models");
+var Models = require("../edit_channel/models");
 var _ = require("underscore");
 require("admin.less");
-var BaseViews = require("edit_channel/views");
-var dialog = require("edit_channel/utils/dialog");
-var stringHelper = require("edit_channel/utils/string_helper");
+var BaseViews = require("../edit_channel/views");
+var dialog = require("../edit_channel/utils/dialog");
+var stringHelper = require("../edit_channel/utils/string_helper");
 var fileDownload = require("jquery-file-download");
+var AdminRouter = require("./router")
+
+function scopeEvents(events, scopeSuffix){
+    // adds a suffix to backbone event selectors (i.e. a class)
+    events = JSON.parse(JSON.stringify(events))
+    let scopeEvents = {}
+    for (let k in events){
+        scopeEvents[k+scopeSuffix] = events[k]
+    }
+    return events
+}
 
 var AdminView = BaseViews.BaseView.extend({
     lists: [],
     template: require("./hbtemplates/admin_area.handlebars"),
     initialize: function(options) {
+        this.router = options.router;
         this.render();
     },
     render: function() {
         this.$el.html(this.template())
+
+        this.user_collection = new Models.UserCollection();
+        this.user_collection.filterOptions = AdminRouter.USER_FILTERS;
```

Comment:
This is only called once, so we shouldn't have to worry about `AdminRouter.USER_FILTERS` (and other defaults) getting mutated.